README.md

```diff
@@ -95,7 +95,6 @@ None
 - `claimForm.showAdjustmentAtEnter`, boolean to display adjustment field at claim entry. Default false;
 - `claimForm.insureePicker`, the insuree picker to use when filling a claim. Default insuree.InsureeChfIdPicker (the exact chfid entry picker). Other pre-canned option (from insuree reference module): insuree.InsureePicker (dialog picker with search on chfid, last name and other names)
 - `canSaveClaimWithoutServiceNorItem`, wherever user can save a claim without service nor item, default: true
-- `canSubmitClaimWithZero`, wherever user can submit with 0 as claimed amount (probably a claim without service/item), default: false
```

Comment:
@delcroip @dragos-dobre please comment

Reply:
@wzglinieckisoldevelo In this context, submitting a claim implies that the user has the ability to TRY to submit a claim. Previously, if any of the selected claims failed to meet the criteria (at least one service/item), attempting to submit a claim was not possible. This was deemed as not user-friendly.

Reply:
You must leave this there. In TZ because of the capitation payment mechanism, the price of services and items is not considered so they have the value 0. This means the total claim amount can be 0. 

As discussed, please allow the submission of all claims and the ones with no items and services will be removed automatically by the rule based engine.

Reply:
Second though, if we allow to submit any claim then the configuration might not be required. Don't know if there is a check if the claim has items and services.

Reply:
@dragos-dobre This was my way of thinking, `canSubmitClaimWithZero ` controlled whether users could try submitting a claim without item/service. Previously, if a claim lacked items/services, it prevented submitting all of selected claims. Now that we permit attempts to submit claims without items/services, keeping this setting seems unnecessary.

Reply:
@dragos-dobre up

Reply:
we should differentiate failing submit != rejected claim

Reply:
I will merge it, @wzglinieckisoldevelo / @hirensoni913 I will create a page/ticket to manage the difference with failing to summit and rejected